index.php

```diff
@@ -42,15 +42,6 @@
 $mageFilename = MAGENTO_ROOT . '/app/Mage.php';
 $maintenanceFile = 'maintenance.flag';
 
-if (!file_exists($mageFilename)) {
-    if (is_dir('downloader')) {
-        header("Location: downloader");
-    } else {
-        echo $mageFilename." was not found";
-    }
-    exit;
```

Comment:
this also removes the exit, if the Mage.php does not exist, which I think is not intended.

Reply:
It was intentional, quoting my first comment:

Also, I think the purpose of checkin if the MAGENTO_ROOT . '/app/Mage.php' is useless, one file check for every request while magento wouldn't work at all if it didn't exist, I think that check has to be removed, there's no point in executing it.